README.md

```diff
@@ -1,12 +1,88 @@
-Chloris
-=======
+# Chloris
+
 
 Web application made for remembering about watering the plants. 
 
-**How to run it**
+## How to run it on Linux
+###Prerequisites
+First, check if git and docker are present on your system. If you do not have it, complete the basic installation using
+the documentation.
+### Installing
+Then clone this repository 
+
+`git clone https://github.com/zawias2704/chloris.git`
+
+Next, go to the project directory and run the following command at the terminal to run docker
+
+`./docker/up`
+
+And run
+
+`./docker/attach`
+
+To install the composer, create a database, and load the sample users, run the command
+
+`phing build`
+
+And run it in your browse of your choice. 
+
+`10.0.0.1`
+
+To quit the application run:
+
+`exit`
+
+And
+
+`./docker/down`
+## Running the tests
+To turn on tests after turning on
+
+`./docker/attach`
+
+Run
 
-First, check if git and php are present on your system. Then clone this repository
+`phing test`
+
+## How to run it on macOS
```

Comment:
Delete this part. Do not copy yourself, Mac configuration does not differ that much.